ci/test_conda.sh

```diff
@@ -9,23 +9,29 @@ set -euo pipefail
 rapids-logger "Starting Conda Package Test"
 
 rapids-logger "Creating Test Environment"
-rapids-mamba-retry create -n test
+# TODO: replace this with rapids-dependency-manager
+rapids-mamba-retry create -n test \
+  click \
+  pytest \
+  clangdev >=18 \
+  cuda-nvcc >=12.5 \
+  cuda-version >=12.5 \
```

Comment:
We probably want something like this:
```suggestion
  cuda-version =${RAPIDS_CUDA_VERSION%.*} \
```

Compare to https://github.com/rapidsai/cudf/blob/03742acc9ae72a4e5a582fb3f136c9116862f062/ci/test_cpp_common.sh#L17